packages/eth-json-rpc-provider/src/index.ts

```diff
@@ -1,3 +1,6 @@
 export * from './provider-from-engine';
 export * from './provider-from-middleware';
-export { SafeEventEmitterProvider } from './safe-event-emitter-provider';
+export {
+  SafeEventEmitterProvider,
+  type Eip1193Request,
```

Comment:
I just realized that we are exporting this type. Is this something that you think would be useful for consumers? I feel that we ought to err on the side of not including imports we may not need at first to keep the surface area of the package smaller.